hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/tools/federation/RouterAdmin.java

```diff
@@ -384,6 +398,13 @@ public int run(String[] argv) throws Exception {
         getDisabledNameservices();
       } else if ("-refresh".equals(cmd)) {
         refresh(address);
+      } else if ("-initViewFsToMountTable".equals(cmd)) {
+        if (initViewFsToMountTable(argv[i])) {
+          System.out.println("Successfully init ViewFs mapping to router " +
+              argv[i]);
+        } else {
```

Comment:
if this command run failed, we should print err information such as `System.err.println("Failed when execute command 'initViewFsToMountTable'")` or something else;